provisioning_templates/snippet/redhat_register.erb

```diff
@@ -47,6 +47,15 @@ snippet: true
 #
 #   http-proxy-password = <password>            Proxy password to be used for registration
 #
+#   syspurpose_role                             Sets the system purpose role
+#
+#   syspurpose_usage                            Sets the system purpose usage
+#
+#   syspurpose_sla                              Sets the system purpose SLA
+#
+#   syspurpose_add_ons                          Sets the system purpose add-ons. Seperate multiple
```

Comment:
Do you have a strong preference for 'addon_ons' vs 'addons' ? I slightly prefer the latter since it maps more closely to the CLI command. 

Also, 'separate' gets me 50% of the time too!